pkg/policy/consumer.go

```diff
@@ -259,25 +280,29 @@ func (c *Consumable) AllowConsumerLocked(cache *ConsumableCache, id NumericIdent
 // consumers map and the given consumable to the given consumer's consumers map.
 // Must be called with Consumable mutex Locked.
 func (c *Consumable) AllowConsumerAndReverseLocked(cache *ConsumableCache, id NumericIdentity) {
-	log.Debugf("Allowing direction %d -> %d", id, c.ID)
+	scopedLog := log.WithFields(log.Fields{
+		logfields.Identity + ".from": c.ID,
+		logfields.Identity + ".to":   id,
+	})
+	scopedLog.Debug("Allowing direction")
 	c.AllowConsumerLocked(cache, id)
 
 	if reverse := cache.Lookup(id); reverse != nil {
-		log.Debugf("Allowing reverse direction %d -> %d", c.ID, id)
+		scopedLog.Debug("Allowing reverse direction")
```

Comment:
This message will have the same `from` and `to` fields, but the direction is opposite, so I think they should be swapped here.

Reply:
Yeah, I remember thinking "well, it says reverse, so I should show the un-reverse" but the old message had them in the other order. I'll fix it.